providers/edge/src/airflow/providers/edge/plugins/templates/edge_worker_hosts.html

```diff
@@ -97,6 +100,25 @@ <h2>Edge Worker Hosts</h2>
                 {% endfor %}
               </ul>
             </td>
+            <td>
+              {%- if host.state == "idle" or host.state == "running" -%}
+                <form action="../edgeworkerhosts/status/maintenance/{{ host.worker_name }}/on" method="GET">
+                  <button type="submit" style="padding: 10px 20px; background-color: blue; color: white; border: none; border-radius: 5px;">
+                      Enable
```

Comment:
Could we call it "Enter"?
```suggestion
                      Enter
```